test/Project.toml

```diff
@@ -0,0 +1,4 @@
+[deps]
```

Comment:
is this file needed?

Reply:
Yeah it's the new post-1.3 way of setting up test dependency.